### ISSUE TITLE
Misc Science was using 400,000 watts I think because i ugpraded the cooler and  freezer

### ISSUE THREAD

**Issue by Kromgar** [tgstation/tgstation#11534]
![image](https://cloud.githubusercontent.com/assets/6778786/9621572/1832b158-50f6-11e5-8348-8c12fe326efd.png)

**Comment by Kromgar** [comment]
THat or all the hacked doors in misc science

**Comment by duncathan** [comment]
You're correct, it was because of the freezer. It's a feature, actually.

**Comment by Iamgoofball** [comment]
holy shit what

i need to fix that

**Comment by duncathan** [comment]
oh sorry I thought it was a feature because "it's ok guys I balanced T4"
although I suppose the fact that they could go into negative temperatures and break the game should've been a clue that perhaps not all was intended there

**Comment by Kromgar** [comment]
Okay its not from upgrading the freezer and heater both use 200k energy

**Comment by Iamgoofball** [comment]
@duncathan Yeah, basically, upgraded heaters doing more power draw was a thing raz had in there. I brought it up about concerns of too much power usage with T4s, but a few people said "Naaah keep it, ghetto power sinks!" so I did

**Comment by GunHog** [comment]
I think this should stay.

**Comment by Lumipharon** [comment]
It's dumb and makes no sense. Also it means people accidentally just fuck the power an have no idea why.

**Comment by Marflowerose** [comment]
I wouldn't mind T4's using lots of power if I could actually supply the power. Since they only use APC and APC can recharge only at a certain pace, the APC goes to 0 even if you have 4 singularitys wired to grid.

**Comment by Actionb** [comment]
It wouldn't be quite so bad if the freezer would shut off when losing power - but as it is, the freezer/heater essentially powersinks the area.
And the amount of power drawn feels off too. A mech bay with fully upgraded chargers and two mechs charging needs around 130kw - which seems reasonable. A single freezer, however, shouldn't draw over twice the station's normal power load.

**Comment by Miauw62** [comment]
tl;dr: our power system sucks

**Comment by Marflowerose** [comment]
Having machines that use massive amount of power require direct power cable would alleviate this. Emitters require direct power, why not heater/cooler?

**Comment by Kromgar** [comment]
![image](https://cloud.githubusercontent.com/assets/6778786/9640524/0c0236ee-5180-11e5-8a6c-d38bd1903d96.png)

**Comment by Actionb** [comment]
That's just 2868 watt + some weird decimals, doofus.

**Comment by Tokiko1** [comment]
Keep in mind that you will turn TEGs and turbines into ridiculous free energy machines if you lower the powerdrain of freezers/heaters.

**Comment by Marflowerose** [comment]
Problem is not the energy usage of Freezer/heater, it's that there's currently no way to provide the machine with the power it requires via APC since there's a cap on how fast APC can recharge. Allow freezer/heater to utilize direct power from cable underneath instead of APC and it'll work just fine.

**Comment by Dorsisdwarf** [comment]
I was under the impression that any machine could be supplied with auxiliary power straight from the grid with a wire knot underneath.

**Comment by blobbernaut** [comment]
That's a good idea, why isn't it like that

**Comment by Marflowerose** [comment]
@Dorsisdwarf I think that only applies to emitters and theoretically to SMES, but SMES are/were so buggy that it's better to just build the terminal and I think it got removed from SMES.
